### PR TITLE
initial-project

### PR DESCRIPTION
**Pull request by kienyk02**
## Related Tickets
- [#Task 78107: Content](https://edu-redmine.sun-asterisk.vn/issues/78107)
## WHAT
- Tạo cấu trúc thư mục MVP.
- Thiết kế giao diện App: Logo, Copyright label.
## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/blob/master/eng/android/coding_convention_android_kotlin.md | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(Other notes)*

### CODE REVIEW

**Review by nhat15072002 — APPROVED** [review]

**Review by nhat15072002 — APPROVED** [review]

**Review by huyenmi16 — APPROVED** [review]

File: app/src/main/java/com/example/anidb/utils/Constant.kt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package com.example.anidb.utils
+
+class Constant {
+}
```
Comment (ghost) [inline app/src/main/java/com/example/anidb/utils/Constant.kt]: End of file's content must be a newline

File: app/src/main/java/com/example/anidb/screen/home/presenter/HomeContract.kt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package com.example.anidb.screen.home.presenter
+
+class HomeContract {
+}
```
Comment (ghost) [inline app/src/main/java/com/example/anidb/screen/home/presenter/HomeContract.kt]:
```suggestion
}

```

File: app/src/main/java/com/example/anidb/screen/home/adapter/AnimeHomeAdapter.kt [hunk 3]
```diff
@@ -0,0 +1,4 @@
+package com.example.anidb.screen.home.adapter
+
+class AnimeHomeAdapter {
+}
```
Comment (ghost) [inline app/src/main/java/com/example/anidb/screen/home/adapter/AnimeHomeAdapter.kt]:
```suggestion
}

```

File: app/src/main/java/com/example/anidb/screen/detail/adapter/AnimeRelateAdapter.kt [hunk 4]
```diff
@@ -0,0 +1,4 @@
+package com.example.anidb.screen.detail.adapter
+
+class AnimeRelateAdapter {
+}
```
Comment (ghost) [inline app/src/main/java/com/example/anidb/screen/detail/adapter/AnimeRelateAdapter.kt]:
```suggestion
}

```

File: app/src/main/java/com/example/anidb/screen/detail/AnimeDetailPresenter.kt [hunk 5]
```diff
@@ -0,0 +1,4 @@
+package com.example.anidb.screen.detail
+
+class AnimeDetailPresenter {
+}
```
Comment (ghost) [inline app/src/main/java/com/example/anidb/screen/detail/AnimeDetailPresenter.kt]:
```suggestion
}

```

File: app/src/main/java/com/example/anidb/data/repository/source/remote/AnimeRemoteDataSource.kt [hunk 6]
```diff
@@ -0,0 +1,4 @@
+package com.example.anidb.data.repository.source.remote
+
+class AnimeRemoteDataSource {
+}
```
Comment (ghost) [inline app/src/main/java/com/example/anidb/data/repository/source/remote/AnimeRemoteDataSource.kt]:
```suggestion
}

```

File: app/src/main/java/com/example/anidb/data/repository/source/local/AnimeLocalDataSource.kt [hunk 7]
```diff
@@ -0,0 +1,4 @@
+package com.example.anidb.data.repository.source.local
+
+class AnimeLocalDataSource {
+}
```
Comment (ghost) [inline app/src/main/java/com/example/anidb/data/repository/source/local/AnimeLocalDataSource.kt]:
```suggestion
}

```

**Review by ghost — CHANGES_REQUESTED** [review]
Check cmts.
Ticket redmine phải để 90%, status Reviewing + điền link Pull Request nhé

File: .github/pull_request_template.md
Comment (ghost) [inline .github/pull_request_template.md:1]: Em không sửa file này nhé!
Mỗi khi em tạo PULL REQUEST, nội dung file này auto được copy vào PR Description rồi, em chỉ sửa Description thôi, không sửa file này!!!

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]
good

**Pull request merged**